src/server.ts

```diff
@@ -406,10 +406,15 @@ export class Server extends EventEmitter {
         if (authResult instanceof Error) {
           return this._sendError({
             Code: {
-              Value: 'SOAP-ENV:Server',
-              Subcode: { value: 'InternalServerError' },
+              Value: 'soap:Receiver',
```

Comment:
so it's been like this for a while, why does this need to be changed?

Reply:
This was not according to soap envelope 1.2 schema: https://www.w3.org/2003/05/soap-envelope/
valid values for Fault->Code->Value are
```
<xs:enumeration value="tns:DataEncodingUnknown"/>
<xs:enumeration value="tns:MustUnderstand"/>
<xs:enumeration value="tns:Receiver"/>
<xs:enumeration value="tns:Sender"/>
<xs:enumeration value="tns:VersionMismatch"/>
```

and the namespace tag "soap" for "https://www.w3.org/2003/05/soap-envelope/" is being declared in src/server.ts line: 589-596 
tag "SOAP-ENV is not being declared anywhere

thus the **soap:Server**

fully schema compliant Saop Envelope 1.2 Fault reponse is like this:
```
<env:Envelope xmlns:env="http://www.w3.org/2003/05/soap-envelope"
              xmlns:m="http://www.example.org/timeouts"
              xmlns:xml="http://www.w3.org/XML/1998/namespace">
 <env:Body>
  <env:Fault>
   <env:Code>
     <env:Value>env:Sender</env:Value>
     <env:Subcode>
      <env:Value>m:MessageTimeout</env:Value>
     </env:Subcode>
   </env:Code>
   <env:Reason>
     <env:Text xml:lang="en">Sender Timeout</env:Text>
   </env:Reason>
   <env:Detail>
     <m:MaxTime>P5M</m:MaxTime>
   </env:Detail>    
  </env:Fault>
 </env:Body>
</env:Envelope>
```

Reply:
What would be nice to have is the possibility to switch to SOAP 1.1 envelope Fault response. It has a different structure. Maybe use the existing forceSoap12Headers option for this? or maybe create a new option?